pallets/subtensor/src/lib.rs

```diff
@@ -1419,8 +1427,14 @@ pub mod pallet {
         StorageMap<_, Identity, NetUid, u64, ValueQuery, DefaultEMAPriceMovingBlocks<T>>;
     #[pallet::storage]
     /// --- MAP ( netuid ) --> global_RAO_recycled_for_registration
-    pub type RAORecycledForRegistration<T> =
-        StorageMap<_, Identity, NetUid, u64, ValueQuery, DefaultRAORecycledForRegistration<T>>;
+    pub type RAORecycledForRegistration<T> = StorageMap<
+        _,
+        Identity,
+        NetUid,
+        TaoCurrency,
```

Comment:
I don't think we need one, but should double check don't need a migration for these